### PR TITLE
Keadm error handling improvements

### LINKED ISSUE #698: keadm should not ignore errors
**What happened**:

in many cases, keadm ignores errors and continues.

**What you expected to happen**:

it should stop processing and prompt the user.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeEdge version: 0.3.0
- Hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by yamt**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug

> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

xref #698 
(partially)

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sids-b** [comment]
@pavan187 @srivatsav123 @subpathdev @kadisi , any idea why CI does not run for installer code ?

**Comment by kadisi** [comment]
perhaps we need check `TRAVIS_COMMIT_RANGE `variable.

```
before_install:
  - |
      echo "TRAVIS_COMMIT_RANGE:"$TRAVIS_COMMIT_RANGE
      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(.md)|(.png)|(.pdf)|(.html)|^(LICENSE)|^(docs)'
      then
        echo "Only doc files were updated, skip running the CI."
        exit
      fi

```

**Comment by subpathdev** [comment]
Yes, the problem is that the commit range is to short. I will create a fix for this problem. 
(First possible solution: comparing to origin/master) 
(Second: finding the commit, which is in origin/master and the current branch and comparing with them)

File: keadm/app/cmd/cloud/init.go [hunk 1]
```diff
@@ -62,13 +61,13 @@ func NewCloudInit(out io.Writer, init *types.InitOptions) *cobra.Command {
 		Short:   "Bootstraps cloud component. Checks and install (if required) the pre-requisites.",
 		Long:    cloudInitLongDescription,
 		Example: cloudInitExample,
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (rohitsardesai83) [inline keadm/app/cmd/cloud/init.go]: why is this renamed as RunE ?

Comment (yamt) [reply keadm/app/cmd/cloud/init.go]: Run and RumE are separate fields of the struct. See the Cobra documentation.
I guess E stands for error.

File: keadm/app/cmd/util/k8sinstaller.go [hunk 2]
```diff
@@ -49,7 +49,8 @@ func (ks *K8SInstTool) InstallTools() error {
 	case types.VersionNAInRepo:
 		return fmt.Errorf("Expected %s version is not available in OS repo", component)
 	case types.AlreadySameVersionExist:
-		return fmt.Errorf("Same version of %s already installed in this host", component)
+		fmt.Println("Same version of %s already installed in this host", component)
```
Comment (rohitsardesai83) [inline keadm/app/cmd/util/k8sinstaller.go]: Same version of %s  *is* already installed .
similarly at other places .

Comment (yamt) [reply keadm/app/cmd/util/k8sinstaller.go]: do you mean to improve the message? i feel it's a separate change.

Comment (rohitsardesai83) [reply keadm/app/cmd/util/k8sinstaller.go]: ok , we can have a separate PR for this.

**Pull request merged**